src/styles/styles.js

```diff
@@ -3959,6 +3959,9 @@ const styles = (theme) => ({
         height: 30,
         width: '100%',
     },
+    versionTextColor: {
+        color: `${Colors.ivory}cc`,
```

Comment:
```suggestion
        color: theme.iconColorfulBackground,
```

The style name may need to be changed to have same signature as other text styles e.g. textBlue, textWhite. I don't know what name would be best here.

Reply:
I have added `textIvoryLight` as a new variable.